explorer-api/src/mix_node/models.rs

```diff
@@ -36,6 +36,7 @@ pub(crate) struct PrettyDetailedMixNodeBond {
     pub avg_uptime: u8,
     pub estimated_operator_apy: f64,
     pub estimated_delegators_apy: f64,
+    pub operating_cost: Coin,
```

Comment:
Isn't the value returned here supposed to be "our" `Coin` as opposed to cosmwasm's?

Reply:
![image](https://user-images.githubusercontent.com/6359431/198571168-01c31694-af35-45d9-a5be-8eb8822e5ddc.png)
it's the case right?